.travis.yml

```diff
@@ -7,12 +7,23 @@ go:
 env:
 - GO111MODULE=on
 
-before_install:
-  - go get github.com/mattn/goveralls
-  - go get github.com/lawrencewoodman/roveralls
-
-script:
-  - make test
-  - make build.docker
-  - roveralls -ignore cmd/e2e
-  - goveralls -v -coverprofile=roveralls.coverprofile -service=travis-ci
+jobs:
+  include:
+    - stage: Build
+      before_script:
+        - go get github.com/mattn/goveralls
+        - go get github.com/lawrencewoodman/roveralls
+      script:
+        - make test
+        - make build.docker
+        - roveralls -ignore cmd/e2e
+        - goveralls -v -coverprofile=roveralls.coverprofile -service=travis-ci
+    - stage: Documentation Tests
+      before_script:
+        - go get sigs.k8s.io/kind
+        - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.12.0/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
```

Comment:
I would use an env for changing Kubernetes version

Reply:
hm.. we're not re-using this anywhere except at this place. Once we do, I'll externalize this into env.